src/components/organisms/offerContact/offerContact.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+import { Map, TileLayer } from 'react-leaflet'
+import { Row, Col } from 'react-bootstrap'
+
+import './map.css'
+
+const OfferContact = ({offer}) => {
+
+  let latlng ={}
+
+  offer.location != null?
+  	latlng = {
+      lat: offer?.location?.latitude,
+      lng: offer?.location?.longitude
+    }
+  :
+  latlng = {
+	lat: 51,
+	lng: 30
+  };
```

Comment:
To można zapisać dużo prościej:

```suggestion
  const position = {
    lat: offer?.location?.latitude || 51,
    lng: offer?.location?.longitude || 30
  }
```

Aczkolwiek raczej bym nie wyświetlał w ogóle mapy jeśli w ofercie brakowałoby z jakiegoś powodu pozycji.